curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -131,47 +131,47 @@ What does the `transform` CSS property allow you to do?
 
 #### --distractors--
 
-Change the color and font style of an element
+Change the color and font style of an element.
 
 ---
 
-Adjust the margin and padding of an element
+Adjust the margin and padding of an element.
 
 ---
 
-Modify the visibility and display of an element
+Modify the visibility and display of an element.
 
 #### --answer--
 
-Rotate, scale, skew, or translate an element
+Rotate, scale, skew, and translate an element.
```

Comment:
A number of these aren't in the review page - but they are in the video, so it's fine.